cpp/net/url_fetcher.cc

```diff
@@ -276,5 +276,41 @@ ostream& operator<<(ostream& output, const UrlFetcher::Response& resp) {
   return output;
 }
 
+ostream& operator<<(ostream& output, const UrlFetcher::Request& req) {
+  output << "verb: " << req.verb << endl
+         << "url: " << req.url << endl
+         << "headers: " << req.headers << endl
+         << "body: <<EOF" << endl
+         << req.body << "EOF" << endl;
+  return output;
+}
+
```

Comment:
Only one line here (and before the preceding definition), but two later on? We should probably pick one for the file... (I vote for two!) :smiley: